tools/packages.yaml

```diff
@@ -8906,6 +8941,11 @@ url: https://github.com/upx/upx/releases/
 activity: low
 ---
 kind: url
+name: uriparser
+url:https://github.com/uriparser/uriparser/releases/
+activity: low
+---
```

Comment:
The yaml "error" is a bit deceiving.  The reason is there needs to be a space after `url:`.

Reply:
Fixed!